New Post: Plugins

MrTronMan opened the pull request.

KorbsStudio commented:
This shouldn't start out with:
```
<h1 id="getting-started-with-plugins">Getting started with plugins</h1>
```
Since the title belongs in the sidebar of the article, although I liked that you used an ID for links and such.

Also, other areas of the sidebar need to be edited (Report, Edit, etc).

Also, I think we can more informational, like fully explaining why users shouldn't have the server running while installing new plugins. Also, explain why users should restart instead of reloading(as this can break plugins like DiscordSRV).
I want our help center to provide as much information as possible, even if it's a lot! 

Also during this update of our help center, we'll like to correct our grammar, I have been using Grammarly for this:
![image](https://i.imgur.com/zA1cbJT.png)

MrTronMan commented:
I have renamed it to plugins instead of installing plugins because this is all about plugins now 😄 

I have added more detail such as a way to turn off the plugin without deleting it. I have also added details on why you should turn off the server and also not reload, I have also moved that more toward the top.

KorbsStudio reviewed (changes requested):
For **Turning off a plugin** and **Turning back on the plugin**, they should both be changed to `<h3>` instead of `<h2>` and should be put under a `<h2>` title called **Enabling and Disabling Plugins**. Of course, changing both of the `<h3>` titles to **Enabling** and **Disabling**.

Example:
# Enabling and Disabling
## Enabling
## Disabling
__ 
Add explanation behind what a YML file is for **Configuration**.

A list of recommendations for plugins might be added in the future, providing alternatives.

MrTronMan commented:
**Updated the plugins.html**

Added, what you said. Added an explanation on `YMLs` (YAMLS). Also decided to rename some of the headings.

KylePlays-Studio commented:
this falix trash yeet polarnodes better than falixnodes yeet

MrTronMan reviewed (commented):
Updated the plugins.html

Added, what you said. Added an explanation on YMLs (YAMLS). Also decided to rename some of the headings.

KorbsStudio commented on article/minecraft/general/plugins/index.html:
```diff
@@ -0,0 +1,57 @@
+                <h3>Meta</h3>
+                <p id="title">Plugins</p>
+                <p id="date">Last updated: March 25th, 2021</p>
+                <p id="category">Minecraft General/p>
```
Syntax error

MrTronMan reviewed (commented):
Moved file locations, as well as I, fixed the syntax error.

KylePlays-Studio commented:
stfu dis surver suks

KylePlays-Studio commented:
stfu dis surver suks

KylePlays-Studio commented:
stfu dis surver suks

KylePlays-Studio commented:
stfu dis surver suks

KylePlays-Studio commented:
stfu dis surver suks

KylePlays-Studio commented:
stfu dis surver suks

MrTronMan commented:
Added more about the plugin server jars, where to get plugin server jars.

MrTronMan commented:
You did a revamp and changed it to Markdown so I will convert and update the repository.
